src/sidebar/components/annotation-action-bar.js

```diff
@@ -10,9 +10,22 @@ import { withServices } from '../util/service-context';
 import AnnotationShareControl from './annotation-share-control';
 import Button from './button';
 
+/** @typedef {import("../../types/api").Annotation} Annotation */
+
+/**
+ * @typedef AnnotationActionBarProps
+ * @prop {Annotation} annotation - The annotation in question
+ * @prop {(Event) => any} onReply - Callbacks for when action buttons are clicked/tapped
```

Comment:
`Event` is interpreted as a parameter name rather than type here. In other words, it behaves like `(Event: any) => any)`. I suggest `(e: Event) => any` instead.